projects/step-frontend/src/lib/modules/execution/components/aggregated-tree-node-history/aggregated-tree-node-history.component.scss

```diff
@@ -0,0 +1,77 @@
+@use 'projects/step-core/styles/core-variables' as var;
+
+$lines-color: var.$gray-300;
+
+.container {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  margin-top: 1rem;
+}
+
+.previous-executions-container {
+  margin-right: 3rem;
+  display: flex;
+  flex-direction: row;
+  gap: 1.6rem;
+}
+
+.no-previous-execution {
+  opacity: 0.8;
+  display: flex;
+  align-items: center;
+}
+
+.empty-piechart {
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  border: 1px solid $lines-color;
+  position: relative;
+  z-index: 1;
+  background: white;
+}
+
+.connect-line::after {
+  content: '';
+  position: absolute;
+  top: 20px;
+  left: 50%;
+  height: 2px;
+  width: 40px;
+  background: $lines-color;
+  transform: translateY(-50%);
+  z-index: -1;
+}
+
+.history-item-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  z-index: 1;
+  --arrow-w: 10px; /* triangle base width */
+  --arrow-h: 6px; /* triangle half-height */
```

Comment:
```suggestion
  --arrow-w: 10px;
  --arrow-h: 6px;
```

Reply:
I removed these, even if normally I'd leave them especially for this visual tricks